Packs/WolkenITSM/Integrations/WolkenITSM/WolkenITSM.yml

```diff
@@ -0,0 +1,310 @@
+      description: ""
+      type: String
+  - arguments:
+    - description: Pass Incident Id
+      name: incidentId
+      required: true
+    - description: Pass the required Resolution code
+      name: Resolution_Code
+      required: true
+    - description: 'Pass Resolution Notes '
+      name: Resolution_Notes
+      required: true
+    - description: Pass Closure description while closing the incident
+      name: Closure_Description
+      required: true
+    - description: 'Pass Status Id of the incident '
+      name: StatusId
+      required: true
+    - description: Pass Sub Status Id of the incident
+      name: SubStatusId
+      required: true
+    - description: Pass Owner of the incident
+      name: Owner
+      required: true
+    description: Close an Incident
+    name: wolken-post-api-v1-incidents-close
+    outputs:
+    - contextPath: Wolken.UpdateIncidents.status
+      description: ""
+      type: String
+    - contextPath: Wolken.UpdateIncidents.message
+      description: ""
+      type: String
+  - arguments:
+    - description: Pass Incident Id
+      name: incidentId
+      required: true
+    - description: Pass description for incident Cancellation
+      name: Description
+      required: true
+    description: Cancel an Incident
+    name: wolken-put-api-v1-incidents-cancel
+    outputs:
+    - contextPath: Wolken.UpdateIncidents.status
+      description: ""
+      type: String
+    - contextPath: Wolken.UpdateIncidents.message
+      description: ""
+      type: String
+  - arguments:
+    - description: Pass limit
+      name: limit
+      required: true
+    - description: Pass offset
+      name: offset
+      required: true
+    - description: Login User Id
+      name: userPsNo
+    - description: Status Id
+      name: statusId
+    - description: Sub Status Id
+      name: subStatusId
+    - description: Team Id
+      name: teamId
+    - description: Unit Id
+      name: unitId
+    - description: Creator Id
+      name: creatorId
+    - description: Requester Id
+      name: requesterId
+    - description: Item Id
+      name: itemId
+    - description: Priority Id
+      name: priorityId
+    - description: Assigned User Id
+      name: assignedUserId
+    - description: Created Time Greater Than Equals
+      name: createdTimeGTE
+    - description: Created Time Less then
+      name: createdTimeLT
+    - description: Updated Time Greater Then Equals
+      name: updatedTimeGTE
+    - description: Updated Time Less Then
+      name: updatedTimeLT
+    - description: Updated By User Id
+      name: updatedByUserId
+    description: Search in the list of incident using any parameters
+    name: wolken-search-incidents-by-params
+    outputs:
+    - contextPath: Wolken.GetIncidents.status
+      description: ""
+      type: String
+  dockerimage: demisto/python3:3.9.8.24399
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
+sourcemoduleid: Incident REST API
```

Comment:
Please add fromversion: 6.0.0

Reply:
fromversion:6.0.0 I have added in the yml file of integration on xsoar local instance. Please let me know if some more changes required from my end.